.github/workflows/update-go-version.yml

```diff
@@ -0,0 +1,31 @@
+name: "[Metrics] Update Go SDK Version Constant"
```

Comment:
also, this should probably be in the tag/publish pipeline

Reply:
Done